.github/workflows/publish-liveness-alpha.yml

```diff
@@ -14,7 +14,7 @@ on:
   # This can be useful for publishing hotfixes quickly to unblock customers,
   # but otherwise, we recommend to use a release branch.
   push:
-    branches: [liveness/alpha]
+    branches: [none]
```

Comment:
Does this disable publishing? Would it be better to just comment the whole file out?

Reply:
I'm down to just comment out the whole file, would be simpler